### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by ChrisTG742**
See https://github.com/knadh/listmonk/issues/851

### CODE REVIEW

File: i18n/de.json [hunk 1]
```diff
@@ -454,10 +454,10 @@
     "settings.smtp.name": "SMTP",
     "settings.smtp.retries": "Wiederholungen",
     "settings.smtp.retriesHelp": "Maximale Anzahl an Wiederholungen, wenn eine Machricht fehlschlägt.",
-    "settings.smtp.sendTest": "Send e-mail",
+    "settings.smtp.sendTest": "E-mail senden",
     "settings.smtp.setCustomHeaders": "Benutzerdefinierten Header verwenden",
-    "settings.smtp.testConnection": "Test connection",
-    "settings.smtp.toEmail": "To e-mail",
+    "settings.smtp.testConnection": "Verbindung testen",
+    "settings.smtp.toEmail": "An E-mail",
```
Comment (an0nfunc) [inline i18n/de.json:460]: Isn't this more like "Empänger E-Mail" than "An E-Mail". That sounds strange. Do you use it like that in normal every-day speech?

Comment (ChrisTG742) [reply i18n/de.json:460]: The question is the context this is used in. If it is ment like "recipient", then I'm with you that should translate to "Empfänger". But I thought is is ment as in the setence "should be sent **to**", then "**An**" fits better here.
Maybe @knadh could shed some light in here?
TIA

Comment (knadh) [reply i18n/de.json:460]: image

This is how it looks on the UI. Sending test message `from email` -> `to e-mail`.

Comment (ChrisTG742) [reply i18n/de.json:460]: Okay, then @an0nfunc is right, I'll fix that.

**Pull request merged**